### ISSUE TITLE
You need to accept the welcome modal first

### ISSUE THREAD

**Issue by IntelligenzaArtificiale** [Soulter/hugging-chat-api#15]
Hello, congratulations for the library.
We are using it to run AUTOGPT and BABYAGI.

But now we have this error:

```shell
{"error":"You need to accept the welcome modal first"}
[Error] Failed to create new conversation. Retrying... (1)
{"error":"You need to accept the welcome modal first"}
[Error] Failed to create new conversation. Retrying... (2)
{"error":"You need to accept the welcome modal first"}
[Error] Failed to create new conversation. Retrying... (3)
{"error":"You need to accept the welcome modal first"}
[Error] Failed to create new conversation. Retrying... (4)
{"error":"You need to accept the welcome modal first"}
[Error] Failed to create new conversation. Retrying... (5)
{"error":"You need to accept the welcome modal first"}
```

We're pretty sure the problem is easily circumvented by making a post request to the new control endpoint: action="/chat/settings"

![image](https://user-images.githubusercontent.com/108482353/236474825-8fe9030f-45ef-49aa-8901-33ceb26b0285.png)

**Comment by Soulter** [comment]
Thank you for your help! I have solved the problem and update a new version `0.0.5.4`.

**Comment by Whitelisted1** [comment]
This thread should be closed as the issue is addressed.
